statsmodels/graphics/gofplots.py

```diff
@@ -22,10 +22,16 @@ class ProbPlot(object):
     Parameters
     ----------
     data : array_like
-        1d data array
-    dist : A scipy.stats or statsmodels distribution
-        Compare x against dist. The default is
-        scipy.stats.distributions.norm (a standard normal).
+        A 1d data array
+    dist : callable
```

Comment:
I think callable is not correct.
We need a distribution with appropriate methods but it doesn't have to be callable, AFAIR

Reply:
All of the referenced distribution are callable. The remainder fo the docstring provides the explanation of what is required. It doesn't make sense to try and stick this into a type, which it isn't.